update/common/scripts/5.4/cleanuntranslatablerelations.php

```diff
@@ -53,12 +53,17 @@
  */
 function parseRelationListIds( $relationListXml )
 {
+    if ( empty( trim( $relationListXml ) ) )
+        return;
```

Comment:
function seems to always have returned array before this, so please stick to that.